src/components/SignupForm.js

```diff
@@ -67,10 +88,18 @@ const SignupForm = ({ className }) => {
               // NOTE: Ensure no form errors
               if (hasFormError) return false
 
-              // NOTE: Ensure checked, redirect otherwise display error
-              isChecked
-                ? window.location.replace(signupForm.redirectUrl)
-                : setHasFormError(!hasFormError)
+              // Set the form error if the TOC is not checked
+              if (!isChecked) {
+                return setHasFormError(!hasFormError)
+              }
+              // NOTE: Ensure toc is checked and hidden sneaky field has not been, redirect
+              else if (isChecked && !isHiddenChecked) {
+                return window.location.replace(signupForm.redirectUrl)
+              }
+              // If the TOCs and sneaky form field have been checked, do nothing. 
```

Comment:
Should we make it seem like they've succeeded somehow? Like showing a 'thank you, we'll send you an invite when your application is accepted' message or something. That way they don't try to work around this because it didn't succeed the first time.

Reply:
I was thinking about this too, but I got lazy about building a page for that... but you're probably right. Would it be ok to do that as a v2 along with that decoy question?